src/test/resources/pages/instructorStudentRecordsPageWithPrivateFeedback.html

```diff
@@ -128,7 +128,7 @@ <h4 class="modal-title">
     <div class="row">
       <div class="col-md-12">
         <br>
-        <div class="student_feedback panel panel-default" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
+        <div class="student_feedback panel panel-default load-feedback-session" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
```

Comment:
@whipermr5 The unrelated changes should be reverted manually? Because if i revert the whole commit, all changes will be reverted (included those which are needed).